moveit_ros/warehouse/warehouse/src/import_from_text.cpp

```diff
@@ -197,14 +199,18 @@ void parseQueries(std::istream& in, planning_scene_monitor::PlanningSceneMonitor
       else if (type == "goal")
         parseGoal(in, psm, cs);
       else
-        ROS_ERROR("Unknown query type: '%s'", type.c_str());
+        RCLCPP_ERROR(LOGGER, "Unknown query type: '%s'", type.c_str());
     }
   }
 }
 
 int main(int argc, char** argv)
 {
-  ros::init(argc, argv, "import_from_text_to_warehouse", ros::init_options::AnonymousName);
+  rclcpp::init(argc, argv);
+  rclcpp::NodeOptions node_options;
+  node_options.allow_undeclared_parameters(true);
+  node_options.automatically_declare_parameters_from_overrides(true);
```

Comment:
long term, it would be good to move away from needing this - the introspection of parameters is very useful.